usaspending_api/common/mixins.py

```diff
@@ -52,13 +52,31 @@ def aggregate(self, request, *args, **kwargs):
                 queryset.annotate(item=group_func(group_field)).values('item').annotate(
                     aggregate=agg_function(agg_field)))
         else:
+            group_expr = self._wrapped_f_expression(group_field)
             # group queryset by a non-date field and aggregate
             aggregate = (
-                queryset.annotate(item=F(group_field)).values('item').annotate(
+                queryset.annotate(item=group_expr).values('item').annotate(
                     aggregate=agg_function(agg_field)))
 
         return aggregate
 
+    _sql_function_transformations = {'fy': IntegerField}
+
+    def _wrapped_f_expression(self, col_name):
+        """F-expression of col, wrapped if needed with SQL function call
+
+        Assumes that there's an SQL function defined for each
+        registered lookup."""
```

Comment:
OK, reenabling that test.  Turns out it was just a poorly written test making it appear not to work...